jobs/telemetry_batch_view.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python
+
+from os import chdir
+from os import environ
+from subprocess import call
+
+options = sum([
+    ["--{}".format(key[4:].lower().replace("_", "-")), value]
+    for key, value in environ.items()
+    if key.startswith("TBV_")
+], [])
+
+call(
+    'git',
+    'clone',
+    environ.get('GIT_PATH', 'https://github.com/mozilla/telemetry-batch-view.git'),
+    '--branch',
+    environ.get('GIT_BRANCH', 'master'),
+)
```

Comment:
@acmiyaguchi re-r? on this call that i just modified to support specifying git path and branch

Reply:
Looks good to me. I've implemented something similar with mozetl ([example](https://github.com/mozilla/telemetry-airflow/blob/master/dags/churn.py#L41) and [implementation](https://github.com/mozilla/python_mozetl/blob/master/bin/mozetl-submit.sh#L78)). It might make sense for me to change the script to be consistent between tbv and mozetl.